artifact.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -ex
+
+if [ -f artifact_go.sh ]; then
+  rm -f artifact_go.sh
+fi
+wget -q -O artifact_go.sh 'https://raw.githubusercontent.com/mdblp/tools/feature/add-platform-project/artifact/artifact_go.sh'
```

Comment:
will need to be reviewed once "Tools" PR is approved (https://github.com/mdblp/tools/pull/28)